infra/test-core2js.rkt

```diff
@@ -0,0 +1,109 @@
+#lang racket
+
+(require "../tools/common.rkt" "../tools/core2js.rkt" "../tools/fpcore.rkt")
+
+(define tests-to-run (make-parameter 10))
+(define test-file (make-parameter "/tmp/test.js"))
+(define fuel (make-parameter 1000))
+
+(define ((eval-fuel evaltor fuel [default #f]) expr ctx)
```

Comment:
Might not be necessary yet, but eventually we ought to move this to a library somewhere.